about.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>About &mdash; Adelaide Nalley's Portfolio</title>        
+        <link rel="stylesheet" href="styles/style.css">
+    </head>
+    <body>
+        <main id="home_container">
+        <nav>
+            <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./portfolio.html">Portfolio</a></li>
+            <li><a href="./hobby-blog.html">Hobbies</a></li>
+            <li class="current-page">About</li>
+            </ul>
+        </nav>
+
+        <header>
+            <h1 class = "title">The Scoop on Me</h1>
+        </header>
+    
+        <section>
+            <br>
+            <p>
+            I'm baby raw denim af succulents whatever farm-to-table vaporware, listicle tofu DIY. Neutra portland iPhone affogato kogi hexagon, etsy pop-up tumeric umami synth lomo. Normcore mixtape stumptown put a bird on it, letterpress organic glossier bicycle rights ramps forage beard single-origin coffee. Fixie XOXO chia, sriracha biodiesel yr chillwave flannel cliche fanny pack vape venmo quinoa meditation brunch. You probably haven't heard of them fam bespoke pinterest vinyl aesthetic.
+            </p>
+            <p>
+            Distillery fixie celiac, narwhal poke tousled readymade. Organic kitsch you probably haven't heard of them umami, small batch synth pickled cornhole plaid freegan. Health goth vexillologist fam, pitchfork readymade polaroid pickled cliche. Pickled typewriter plaid bicycle rights artisan food truck meditation palo santo viral fingerstache heirloom put a bird on it adaptogen intelligentsia chicharrones. Cornhole stumptown intelligentsia chicharrones, knausgaard live-edge pabst williamsburg microdosing listicle meditation street art.
+            </p>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
```

Comment:
one cool trick we can do instead of adding all these `<br/>` is giving the `<p>` element extra bottom margin instead